pkg/tests/apis/dashboard/integration/api_validation_test.go

```diff
@@ -789,6 +761,9 @@ func createDashboardObject(t *testing.T, title string, folderUID string, generat
 			"kind":       dashboardv1alpha1.DashboardResourceInfo.GroupVersionKind().Kind,
 			"metadata": map[string]interface{}{
 				"generateName": "test-",
+				"annotations": map[string]interface{}{
+					"grafana.app/grant-permissions": "default",
```

Comment:
Why do we need to specify it here? What's the effect specifying it on the consumer side?

Reply:
Mode4 will not set the default permissions if the client doesn't add this annotation. When going straight to /apis, we only want the default permissions to be created when the request comes from the UI and the annotation is present. If it's not present, then the default permission won't be added.

Reply:
Oh, that was news to me, thanks!

Will this be documented somewhere?